pkgs/development/libraries/cpptest/default.nix

```diff
@@ -0,0 +1,17 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "cpptest-1.1.2";
+
+  src = fetchurl {
+    url = "http://downloads.sourceforge.net/project/cpptest/cpptest/${name}/${name}.tar.gz";
+    sha256 = "09v070a9dv6zq6hgj4v67i31zsis3s96psrnhlq9g4vhdcaxykwy";
+  };
+
+  meta = with stdenv.lib; {
+    homepage = http://cpptest.sourceforge.net/;
+    description = "CppTest is a portable and powerful, yet simple, unit testing framework for handling automated tests in C++";
```

Comment:
Remove pkg name from the description. And try to make it a little bit smaller. Thank you!